bundles/AdminBundle/Controller/Admin/Document/NewsletterController.php

```diff
@@ -89,6 +89,8 @@ public function getDataByIdAction(Request $request): JsonResponse
             'interfaces' => array_values(class_implements($email))
         ];
 
+        $this->addAdminStyle($email, Pimcore\Event\Admin\ElementAdminStyleEvent::CONTEXT_EDITOR, $data);
```

Comment:
Let's use `use` and not the fully qualified class name, to make it a bit shorter ;)